Gemfile

```diff
@@ -30,7 +30,8 @@ gem "bcrypt", "~> 3.1"
 gem "delayed_job", "~> 4.1"
 gem "delayed_job_active_record", "~> 4.1"
 gem "feedbag", "~> 0.9.5"
-gem "feedjira", "~> 2.1.3"
+gem "feedjira", "~> 3.0"
+gem "httparty"
```

Comment:
It's not a huge deal, but I wonder if we'd be better off using Ruby's built in `Net::HTTP` library rather than adding a new dependency. I know `Net::HTTP` doesn't have the greatest reputation for usability, but we're only making a couple of requests so it doesn't seem like it'll make a huge difference. I'll leave it up to you, but my bias is towards minimizing dependencies unless they save us a ton of time and headache.